spec/subscribe_integration_spec.rb

```diff
@@ -9,6 +9,8 @@
       @product = Z::Product.where(:name => "Awesome Product").first ||
         Z::Product.create!(
           :name => "Awesome Product",
+          :features__c => "Now with even more awesome features!",
```

Comment:
I'm fine with removing them in the end. I just needed to add them to work with our specific Zuora settings. I'll make a note to remove them before the next pull request.